### PR TITLE
Changed tooltip to expand all keybinding in OS X

### PR DESCRIPTION
**Pull request by taylankasap**
Expand all keybinding is Cmd+click in OS X

### CODE REVIEW

**Comment by stof** [comment]
:+1:

**Comment by taylankasap** [comment]
Though I failed to comply with pull request standards it's a small change, so you can just fix it with another commit. Sorry for the inconvenience.

**Comment by fabpot** [comment]
ping @nicolas-grekas 
Probably needs to be merged in 2.6 or 2.7?

File: src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php [hunk 1]
```diff
@@ -119,6 +119,9 @@ protected function getDumpHeader()
     addEventListener = function (e, n, cb) {
         e.addEventListener(n, cb, false);
     };
+    
+var isMac = navigator.platform.toUpperCase().indexOf('MAC') >= 0;
```
Comment (nicolas-grekas) [inline src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php]: isMac intermediary variable can be removed by merging this with the ternary check on the line below.
I'd also merge this new `var` declaration with the one above, eg. aftter `idRx`

**Comment by nicolas-grekas** [comment]
Thank you @taylankasap, will be merged in #14855

**Pull request closed without merging**